### DISCUSSION TITLE
Why there are unreleased tags?

### DISCUSSION THREAD

**Discussion by claudioweiler** [keycloak/keycloak#33148] [Q&A] [answered]
Why there are unreleased tags?

e.g. [24.0.8](https://github.com/keycloak/keycloak/releases/tag/24.0.8)

**Comment by jonkoops** [comment, marked as answer]
These are tags to facilitate [Red Hat build of Keycloak](https://access.redhat.com/products/red-hat-build-of-keycloak). They are not intended for regular Keycloak releases.
